### ISSUE TITLE
Proposal: Add definitions for Individuals

### ISSUE THREAD

**Issue by goneall** [spdx/spdx-3-model#299]
To implement NONE and NOASSERTION values for complex types, we need a way to define Individual values.

An individual value is an instance of a Class with specified properties.

I would propose adding these into an Individuals folder.

The file format would be similar to the Vocabularies file format.

Below are proposed sections:

- Summary
- Description
- Metadata
- name within Metadata
- id within Metadata (the ID would be the namespace) followed by a unique ID
- type within Metadata
- propertyValues - any properties with pre-defined values.  Currently, I only expect the ID's to be xsd:string types although we may need to extend these in the future.

**Comment by goneall** [comment]
@zvr - let me know any thoughts - we can implement this after RC

**Comment by zvr** [comment]
Sure.

Not even sure "name" is needed, to be honest.

Here's an example:
```
SPDX-License-Identifier: Community-Spec-1.0

# NoneCopyrightText

## Summary

Individual representing an assertion that no copyright text is present.

## Description

A NoneCopyrightText is the value that is used to indicate
that a software artifact does not contain any copyright information.

## Metadata

- id: NoneCopyrightText
- type: CopyrightText

## Property Values

- text: "NONE"
```

**Comment by goneall** [comment]
> Not even sure "name" is needed, to be honest.

Agree - in fact for our use cases, we probably only need ID and Type.

**Comment by goneall** [comment]
Fixed in PR #456
